libwallet/assets/btc/wallet.go

```diff
@@ -516,3 +516,28 @@ func (asset *Asset) AccountXPubMatches(account uint32, xPub string) (bool, error
 
 	return acctXPubKey.AccountPubKey.String() == xPub, nil
 }
+
+// GetWalletBalance returns the total balance across all accounts.
+func (asset *Asset) GetWalletBalance() (*sharedW.Balance, error) {
+	if !asset.WalletOpened() {
+		return nil, utils.ErrBTCNotInitialized
+	}
+
+	var totalBalance, totalSpendable, totalImmatureReward int64
```

Comment:
I think we can move this to just above the for loop and use just `=`.

Reply:
why just `=`

Reply:
Because there’s no previous value to retain?

Reply:
there is, the balances are summed up with each iteration